### ISSUE TITLE
Disabled button contrast

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#425]
The documentation theme (`soft`) uses colours for `disabled` buttons that may offer insufficient contrast. This requires verification.

**Comment by ppvg** [comment]
This issues persists in the `basic-bold` theme. However, we have previously discussed the option of removing support for `disabled` controls altogether (given their negative impact on accessibility). As such I propose we address it via #950 instead.

**Comment by HeleenSG** [comment]
Closed because we're solving this by removing the disabled examples and state.
